.changelog/unreleased/api-breaking/provider/2171-permissionless-ICS.md.md

```diff
@@ -0,0 +1,36 @@
+
+- Deprecate the `chain-id` parameter in favour of `consumer-id` for all transactions(TXs) and queries targeting a unique consumer chain. Below is a list highlighting the changes in the CLI commands. All commands assume the prefix `interchain-security-pd tx|q provider`.
+  - **TXs:**
+    | Command                                 | Message                             |
+    |-----------------------------------------|-------------------------------------|
+    | `assign-consensus-key [consumer-id] [consumer-pubkey]`  | [MsgAssignConsensusKey](https://github.com/cosmos/interchain-security/blob/feat/permissionless/proto/interchain_security/ccv/provider/v1/tx.proto#L256)             |
```

Comment:
Wrong line. Also, use v5.2.0 as a stable version for the link.

Reply:
v5.2 doesn't have the new msgs definitions, i.e. MsgCreateConsumer. All links have to be fixed once we get a stable version with Permissionless.

Reply:
Changed to a main commit hash.